src/Init/Init_FFTW.cpp

```diff
@@ -124,34 +124,36 @@ void End_FFTW()
 
 
 //-------------------------------------------------------------------------------------------------------
-// Function    :  Patch2Slab_Rho
-// Description :  Patch-based data --> slab domain decomposition (for density)
+// Function    :  Patch2Slab
+// Description :  Patch-based data --> slab domain decomposition
 //
-// Parameter   :  RhoK           : In-place FFT array
-//                SendBuf_Rho    : Sending MPI buffer of density
-//                RecvBuf_Rho    : Receiving MPI buffer of density
+// Parameter   :  VarS           : Arrary of target variable for FFT
```

Comment:
I will say `S` means `Slab`, following some names of parameter (e.q. `SSize` and `SIdx`).
Not `VarK` because I think in this function, we only prepare the data in slab, not in the k-space yet.
Now I name the `Var` in line 212 as `VarPatch`.